CDP4CrossViewEditor/ViewModels/ThingSelectorViewModel.cs

```diff
@@ -252,99 +197,19 @@ private void ExecuteMoveUp()
 
         /// <summary>
         /// Executes move items command <see cref="MoveItemsDown"/>
-        /// Move selected <see cref="Thing"/> one position down inside selection list
+        /// ExecuteMove selected <see cref="Thing"/> one position down inside selection list
         /// </summary>
-        private void ExecuteMoveItemsDown()
+        private void ExecuteMoveDown()
```

Comment:
Q4: Do we still need this method for manual moving since we're now using grids?

Reply:
R4: No.